guide/10-mapping-and-visualization-alt/part7_utility_functions_for_the_map_widget.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<p>In Part 7 of this guide series, we discussed the utility functions of the MapView object, particularly in regard to using the callback and display_message mechanisms in creating interactivity for users. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Need to change Part 7 to 6 after renaming</p>